packages/twenty-server/src/engine/metadata-modules/field-metadata/field-metadata.service.ts

```diff
@@ -125,6 +125,13 @@ export class FieldMetadataService extends TypeOrmQueryService<FieldMetadataEntit
         fieldMetadataInput.options = generateRatingOptions();
       }
 
+      if (fieldMetadataInput.type === FieldMetadataType.LINK) {
+        throw new FieldMetadataException(
+          '"Link" field types are being deprecated, please use Links type instead',
+          FieldMetadataExceptionCode.INVALID_FIELD_INPUT,
```

Comment:
this will throw an internal server error due to an issue we have with error interception @thomtrp @Weiko